linux_os/guide/system/software/system-tools/package_libcap-ng-utils_installed/rule.yml

```diff
@@ -20,6 +20,9 @@ identifiers:
     cce@rhel7: 82980-4
     cce@rhel8: 82979-6
 
+references:
+    srg: SRG-OS-000445-GPOS-00199
```

Comment:
I saw Steve's page when I was working this rule.  I really did not know where to put it, but decided that SRG-OS-000445-GPOS-00199 was the best place because of this line:
"The included utilities are designed to let admins and developers spot apps from various ways that may be running with too much privilege."  I equated that to properly running from the SRG.  I agree that it more aligns with trusted boot.  Is there a better place to map this OSPP rule to?

Reply:
There are a number of packages that are installed in order to assess the security posture of the endpoint. There is no good substitute for netcap or pscap. They are there so that the lab can verify use of least privileges.

Reply:
@stevegrubb @tedbrunell From the perspective of a lab, installing this makes sense. What about for operational systems? Highlights some of the difference between the lab-focused common criteria profile and the operator-focused STIG.

Reply:
we install many things to allow inspection of systems such as ps, ifconfig, lsof, netstat, etc. The difference is that libcap-ng utilities identify what applications present risk because of elevated privileges. Each of the files is about 20k in size.

Reply:
Kept the SRG mapping, but @tedbrunell agrees there may be little need to enable this check in DoD STIG profile. Created https://github.com/ComplianceAsCode/content/issues/5016 to track that conversation.